conf/broker.conf

```diff
@@ -37,8 +37,11 @@ webServicePortTls=8443
 # Enable the WebSocket API service in broker
 webSocketServiceEnabled=false
 
-# Control whether to bind directly on localhost rather than on normal hostname
-bindOnLocalhost=false
+# Hostname or IP address the service binds on, default is InetAddress.getLocalHost().getHostName().
+bindAddress=
+
+# Hostname or IP address the service advertises to the outside world. If not set, the value of bindAddress is used.
+advertisedAddress=
```

Comment:
If the bind address was `0.0.0.0`, then defaulting to bind address might not work. Should we default to hostname for `advertisedAddress` ?